uniflow/flow/extract/extract_md_flow.py

```diff
@@ -18,7 +18,7 @@ def __init__(self) -> None:
         """Extract md Flow Constructor."""
         super().__init__()
         self._extract_md_op = ExtractTxtOp(name="extract_md_op")
-        self._split_md_op = MarkdownHeaderSplitter(name="process_md_op")
+        self._split_md_op = MarkdownHeaderSplitter(name="split_md_op")
```

Comment:
You should consider to use [SplitterOpsFactory](https://github.com/CambioML/uniflow/blob/main/uniflow/op/extract/split/splitter_factory.py) here. Otherwise, your config splitter will have no effect. Here is an example for [PDF flow](https://github.com/CambioML/uniflow/blob/main/uniflow/flow/extract/extract_pdf_flow.py#L38)